pysher/connection.py

```diff
@@ -134,12 +134,12 @@ def _on_open(self):
         self.send_ping()
         self._start_timers()
 
-    def _on_error(self, error):
+    def _on_error(self, ws, error):
         self.logger.info("Connection: Error - %s" % error)
         self.state = "failed"
         self.needs_reconnect = True
 
-    def _on_message(self, message):
+    def _on_message(self, ws, message):
```

Comment:
I don't think it'd make sense to use `*args` here since websockt-client does not pass `message` as a keyword argument so it'll have to be the 3rd argument.

We could do something like:
```py
def _on_message(self, *args):
  message = args[1]
```

But does that really make any difference?

Reply:
https://github.com/websocket-client/websocket-client/blob/dfe6838d7ce12d5037e58a8c480fc0cbf155a802/websocket/_app.py#L280

Reply:
You're right - in this case it doesn't. In the cases where we don't use any parameters, *args is fine though.